.circleci/continue_config.yml

```diff
@@ -412,6 +412,8 @@ jobs:
       JDK8_HOME: /usr/lib/jvm/java-8-openjdk-amd64
     steps:
       - *checkout_project_root
+      - gcp-cli/install
```

Comment:
there was an idea to set this thing up with GCS, now for the life of mine I can't exactly remember the reasoning... nonetheless the connection is already used for BigQuery and we have it set up here, and it works. No point in removing it and in case we want to test something with GCS we have template